src/Illuminate/Cache/Repository.php

```diff
@@ -490,7 +490,7 @@ protected function getMinutes($duration)
             $duration = Carbon::now()->diffInSeconds(Carbon::instance($duration), false) / 60;
         }
 
-        return $duration > 0 ? $duration : null;
+        return (int) ($duration * 60) > 0 ? $duration : null;
     }
```

Comment:
It means if duration is less than 1 second, don't bother store, because we don't know what is it going to do!